app/src/main/java/fr/free/nrw/commons/db/Converters.java

```diff
@@ -0,0 +1,71 @@
+package fr.free.nrw.commons.db;
+
+import android.net.Uri;
+
+import androidx.room.TypeConverter;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+import org.wikipedia.json.GsonUtil;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+
+import fr.free.nrw.commons.location.LatLng;
+
+public class Converters {
+    private static Gson gson= GsonUtil.getDefaultGson();
```

Comment:
I was hoping to see the use of the CommonsApplicationComponent. This has the same end result but is more brittle